azure_pipelines/automation.yml

```diff
@@ -61,18 +61,20 @@ jobs:
     displayName: 'Install xcpretty'
 
 
-  - task: DownloadSecureFile@1
-    name: configuration
-    displayName: 'Get Automation conf.json'
+  - task: AzureCLI@2
```

Comment:
Why not use Azure KeyVault ADO task to get the file?

Reply:
Just try to reuse the script we have otherwise we will create new logic to handling the returned secrets and convert back to conf.json. Both approach relies on the same service connection.